tile.yml

```diff
@@ -71,7 +71,120 @@ forms:
     type: text
     label: Client key
     optional: true
-
+- name: enable_disable
```

Comment:
If it's not already, is it possible to have this file generated using the data in the various brokers/{service_name}/definition.go files? Seems like it would save potential manual maintenance in the future.

Reply:
This is something I'd like to look at. I think we might be able to generate the service plan forms from the same service definitions we already have coded in the registry (adding one more set per-service for the user-configurable stuff).

These forms I'd really like to be able to do in a similar way, but also generate the documentation for `docs/customization.md` because not everyone is using this as an installed tile and those people need docs.

I think my general approach is going to be to write a separate config package that wraps viper and can keep track of/generate docs at the same time.